analysis/utils/process_rates.R

```diff
@@ -66,14 +66,20 @@ subgroups_rates_std[[which(names(subgroups_rates_std) == "smoking_status_comb")]
     TRUE ~ NA_character_
   ))
 # IMD
+# in IMD rankings, 1 is most deprived 32844 is least deprived
+# these subcategories are 1 = 1-32844/5, 2 = 32844/5-32844*2/5, etc.. 
+# so 1 is most deprived (5 on the deprivation scale)
+# and 5 is least deprived (1 on the deprivation scale)
+# https://assets.publishing.service.gov.uk/government/uploads/system/
+# uploads/attachment_data/file/853811/IoD2019_FAQ_v4.pdf
```

Comment:
As discussed I would suggest changing the variable name to make it clear this is different to the usual OpenSafely IMD quintile order.